src/WebJobs.Script.WebHost/Configuration/HttpOptionsSetup.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Configuration
+{
+    public class HttpOptionsSetup : IConfigureOptions<HttpOptions>
+    {
+        internal const int DefaultMaxConcurrentRequests = 100;
+        internal const int DefaultMaxOutstandingRequests = 1000;
+        private readonly IEnvironment _environment;
+
+        public HttpOptionsSetup(IEnvironment environment)
+        {
+            _environment = environment;
+        }
+
+        public void Configure(HttpOptions options)
+        {
+            if (_environment.IsDynamic())
+            {
+                // when running under dynamic, we choose some default
+                // throttle settings.
+                // these can be overridden by the user in host.json
+                if (_environment.IsAppServiceEnvironment())
```

Comment:
Is this setup going to apply configuration from `IConfiguration` allowing this to be set from `host.json`?

Reply:
Yes - these are http extension host.json settings. We default them early in the pipeline, and any user config comes later and can override